plugins/experimental/http_stats/README.md

```diff
@@ -0,0 +1,16 @@
+This is a remap-based origin-intercept plugin that exposes the internal Traffic Server stats. This was based on the stats_over_http plugin, but removes all the features that are natively available in remap (ACLs) or other remap-plugins (eg compression).
```

Comment:
I thought we were deprecating READMEs, documenting in sphinx.

Reply:
Yeh, needs a doc page I think (not sure how strict we are on those rules though). TPTBC can decide :).

Reply:
Im +1 on a docs page, its the first place both our SREs and I go for info on how to use a plugin

Reply:
Docs added. readme removed.